client/lcd/root.go

```diff
@@ -62,21 +66,28 @@ func ServeCommand(cdc *codec.Codec) *cobra.Command {
 			})
 
 			var cleanupFunc func()
+
+			// TODO: re-enable insecure mode once #2715 has been addressed
 			if viper.GetBool(flagInsecure) {
-				listener, err = tmserver.StartHTTPServer(
-					listenAddr, handler, logger,
-					tmserver.Config{MaxOpenConnections: maxOpen},
+				fmt.Println(
+					"Insecure mode is temporarily disabled, please locally generate an " +
```

Comment:
the SSL cert is generated automatically if none is passed as an argument

Reply:
Ah, excellent, let's update the help text.